python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint_search.py

```diff
@@ -883,21 +1007,33 @@ def _boresch_restraint_BSS(
         temperature : :class:`System <BioSimSpace.Types.Temperature>`
             The temperature of the system
 
-        ligand_selection_str : str
+        lig_selection_str: str
             The selection string for the atoms in the ligand to consider
             as potential anchor points.
 
-        receptor_selection_str : str
+        recept_selection_str: str
```

Comment:
Might be good to keep the full description? We are not in the 8 character fortran age any more.

Reply:
Good point! Will do.